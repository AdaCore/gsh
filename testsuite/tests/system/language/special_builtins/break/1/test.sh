for j in `seq 1 10`; do
    break
    echo "FAILED"
done

echo "PASSED"
for j in `seq 1 10`; do
    echo "Entering inner loop"
    for k in `seq 1 10`; do
         break 2
         echo "FAILED"
    done
    echo "FAILED"
done

for j in `seq 1 2`; do
    echo "Entering inner loop"
    for k in `seq 1 10`; do
         break 1
         echo "FAILED"
    done
    echo "Exiting inner loop"
done
echo $?

for j in `seq 1 10`; do
    break 10
done
echo $?

for j in `seq 1 10`; do
     break -1 2>/dev/null
done
echo $?

(for j in `seq 1 10`; do
    break a 2>/dev/null
done
echo "FAILED" 
echo $?
)
true
