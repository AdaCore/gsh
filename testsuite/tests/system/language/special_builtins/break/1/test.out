PASSED
Entering inner loop
Entering inner loop
Exiting inner loop
Entering inner loop
Exiting inner loop
0
0
1
